app/src/main/java/com/elavon/converge/activities/VirtualTerminalService.java

```diff
@@ -305,7 +305,11 @@ private Transaction getTransaction(
         final TransactionAmounts amounts = new TransactionAmounts();
         amounts.setCurrency("USD");
         amounts.setOrderAmount(payment.getAmount());
-        amounts.setTransactionAmount(payment.getAmount() + payment.getTipAmount());
+        if(payment.isAuthzOnly()){
```

Comment:
Why are you doing this for auth only? Here is what the Jira says.. "When we perform a SALE transaction with Tip."

Shouldn't you check if there actually is a tip?